### PR TITLE
[webui] second bugfix for pr #150 and refactor BrowserStore

### PR DESCRIPTION
**Pull request by ralder**
* fixed bug in BrowserStore.actionOnItemsWithPrefix (BrowserStore.getItem wasn't accessible inside it)

* fixed missing require(BrowserStore.jsx) in components/edit_post_modal.jsx

* incapsulate JSON transform into BrowserStore
(it has some overhead because all simple values will be transform to JSON and to avoid this overhead a bit i changed stored place for often using values current_user_id and current_channel_id. Now it are as part of corresponding objects, anyway after reload page they will be reassigned, so saving it in localStore not effective))

P.S. dynamically 'require' for UserStore.jsx in BrowserStore.jsx  is caused to avoid error with circular dependencies these modules

### CODE REVIEW

File: web/react/stores/browser_store.jsx [hunk 1]
```diff
@@ -1,85 +1,104 @@
...
+            localStorage.setItem("local_storage_version", BROWSER_STORE_VERSION);
+        }
+        this._initialized = true;
+    },
 
-module.exports.setGlobalItem = function(name, value) {
-    if (!_initialized) _initialize();
-	localStorage.setItem(name, value);
-};
+    getItem: function(name, defaultValue) {
```
Comment (crspeller) [inline web/react/stores/browser_store.jsx]: getItem and setItem should work the same way as their corresponding global functions. So they should stringify and parse.

Comment (ralder) [inline web/react/stores/browser_store.jsx]: `getItem` and `setItem` use inside self their corresponding global functions therefore they  JSON-transforms values also.

Comment (crspeller) [inline web/react/stores/browser_store.jsx]: Ah, I see. Looks good.

Comment (ralder) [inline web/react/stores/browser_store.jsx]: And what about `current_user_id` and `current_channel_id`? Is really no reason to save them in localStorage?

**Pull request merged**